### DISCUSSION TITLE
Great work guys

### DISCUSSION THREAD

**Discussion by gaspachocheese** [nxthat/nanocl#525] [General]
Created github account just to comment, found ur project through linkedin.
looks amazing, lots of work that was put into it.
is it possible to install kubernetes in nanocl? would be funny

**Comment by 0xle0ne** [comment]
Hey @gaspachocheese thanks for your support!
You can definitly deploy kubernetes inside nanocl, it's up to you to choose to use container or virtual machine to do so!
Deploying nanocl inside nanocl is also possible if you wanna push the fun further!
